CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Optimizely C# SDK Changelog
 ## [Unreleased]	
```

Comment:
Would it be cool if we removed this `[Unreleased]` H2.  We rarely update the CHANGELOG between versions.

Reply:
it has always been here, so I left it as it is.